shared/common-adapters/list.desktop.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+import React, {PureComponent} from 'react'
+import ReactList from 'react-list'
+import {globalStyles} from '../styles'
+
+import type {Props} from './list'
+
+class List extends PureComponent<void, Props<*>, void> {
+  _itemRender = index => {
+    const item = this.props.items[index]
+    return this.props.renderItem(index, item)
+  }
+
+  render() {
+    return (
+      <div
```

Comment:
The nested divs is so you can plunk this down in a flex box and it'll take up the rest of the room and scroll